cleanlab/internal/label_quality_utils.py

```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
 
     Parameters
```

Comment:
These changes do not match the comments I left on this file in your other PR here:
https://github.com/cleanlab/cleanlab/pull/473/files#diff-01ba518873fbd9bb46ff9e13a2af9adc756c202b723dadfa885de9a5327b81f8

You'll want to carefully look through those.

Reply:
I have made the necessary changes for you as "suggestions" so all you need to do now is commit each of these suggestions. But in the future, you want to be more precise when addressing comments our team makes! 
Remember all the little details really matter for high-quality code

Reply:
Whoops, my bad. Fixed it.

Reply:
thanks this PR is good to go!